src/containers/OrdinalsArea.js

```diff
@@ -206,6 +209,43 @@ const OrdinalsArea = ({ className, space }) => {
                             )}
                         </button>
                     </div>
+                    <div className="col-lg-1 col-md-1 col-sm-1 col-2">
```

Comment:
We can create a function in order to keep clean the onClick.
like
`onClick={filterUtxos}`

Reply:
+1